features/share/graphql/graphql-enhancements/python3/requirements.txt

```diff
@@ -0,0 +1,5 @@
+click
+gql
```

Comment:
Instead of requiring `requests` and `requests_toolbelt` please use the gql install group `gql[requests]`